certif/app/templates/components/routes/authenticated/sessions/session-form.hbs

```diff
@@ -40,9 +45,10 @@
                       onChange=(action onDatePicked)
                       dateFormat='d/m/Y'
                       locale='fr'
+                      minDate='today'
```

Comment:
A priori, si on modifie la session après son déroulement, le lendemain par exemple, la minDate sera celle du lendemain.
Du coup, en accord avec Emmy, il faudrait voir si on peut mettre un minDate='today' à condition que this.isNewSession